registration.html

```diff
@@ -0,0 +1,93 @@
+<style>
```

Comment:
The registration page will not have the navigation bar. Also, it should open in another tab.

Reply:
Also, rename this file as `registration-partial.html`